packages/wpcom-proxy-request/src/index.js

```diff
@@ -114,7 +101,9 @@ const makeRequest = ( originalParams, fn ) => {
 	params.callback = id;
 	params.supports_args = true; // supports receiving variable amount of arguments
 	params.supports_error_obj = true; // better Error object info
-	params.supports_progress = supportsProgress; // supports receiving XHR "progress" events
+	// Are HTML5 XMLHttpRequest2 "progress" events supported?
+	// See: http://goo.gl/xxYf6D
+	params.supports_progress = !! window.ProgressEvent && !! window.FormData;
```

Comment:
Alternatively, we could just hardcode `supports_progress` to `true`. Feature detecting this was relevant in 2012, but today all supported browsers safely support XHR upload progress. See also #79245 which removed `progress-event` polyfill from this package.

The `params.supports_progress` field must be there, the iframe proxy code is checking it.

Reply:
Makes sense and is also consistent with the other hardcoded flags 👍 I'll make the change.